meta/documents/user_guide/de/settings_guide.md

```diff
@@ -0,0 +1,175 @@
+# Wayfair Plugin: Einstellungen für Wayfair Markt
+
+Das Wayfair-Plugin enthält eine Sammlung von Einstellungen zur Steuerung des Plugin-Verhaltens.
+Diese Einstellungen sollten erst konfiguriert werden, nachdem die [Autorisierungseinstellungen für das Plugin](initial_setup.md#1-autorisieren-des-wayfair-plugins-für-den-zugriff-auf-wayfair-schnittstellen) für das aktive Plugin-Set konfiguriert wurden.
+
+## Öffnen der Einstellungsseite
+
+So suchen Sie die Einstellungen:
+
+1. Melden Sie sich bei Plentymarkets als Benutzer mit Administratorrechten an
+
+2. Klicken Sie in der oberen Navigationsleiste von Plentymarkets auf `Einrichtung`
+
+    ![Setup button](../../../images/de/menu_setup.png)
+
+3. Klicken Sie in der linken Navigationsleiste der Benutzeroberfläche auf `Märkte`
+
+    ![Markets button](../../../images/de/wayfair_settings/menu_markets.png)
+
+4. Klicken Sie in der Liste der Märkte auf `Wayfair`. Es wird möglicherweise unten angezeigt, anstatt alphabetisch geordnet zu sein.
+
+5. Klicken Sie unter `Wayfair` auf `Home`
+
+    ![home button](../../../images/de/wayfair_settings/menu_markets_wayfair_home.png)
+
+6. Sie können jetzt die Wayfair-Navigationsleiste verwenden, um eine Einstellungsseite wie [`Lager`](#die-seite-lager) auszuwählen.
+
+    ![wayfair nav bar](../../../images/de/wayfair_settings/wayfair_nav_bar.png)
+
+## Die Home Seite
+
+Die Home Seite wird durch Klicken auf `Home` im Menü auf der linken Seite oder durch Klicken auf `Wayfair` in der Wayfair-Navigationsleiste aktiviert. Derzeit werden keine Informationen bereitgestellt.
+
+## Die Lager Seite
+
+Die Lager Seite wird verwendet, um die Lager, die der Lieferant in Plentymarkets verwendet, mit den Wayfair Lieferanten IDs zu verknüpfen, die dem Lieferanten ausgestellt wurden. Die Zuordnungen werden vom Wayfair-Plugin verwendet, wenn es Inventar an Wayfair meldet und Wayfair-Bestellungen verarbeitet, die bei Plentymarkets eingehen.
+
+### Hinweise zu mehreren Lagern pro Wayfair-Lieferanten-ID
+
+Die Topographie des Plentymarkets-Systems stimmt möglicherweise nicht mit der Anzahl der Wayfair Lieferanten-IDs überein. Es ist akzeptabel, eine Wayfair-Lieferanten-ID für mehr als eines der Plentymarkets-Lagerhäuser zu verwenden. **Dies kann jedoch zu unerwartetem Verhalten führen. Beispielsweise kann das Wayfair plugin möglicherweise nicht das richtige Lager für eingehende Bestellungen ermitteln.** -Weitere Informationen finden Sie unter https://github.com/wayfair-contribs/plentymarkets-plugin/issues/92.
+
+Überprüfen Sie bei der Auswahl einer Beziehung zwischen Lagern und Lieferanten ID, die nicht eins zu eins übereinstimmt, das Verhalten der Einstellung [Lagerpuffer] (#Lagerpuffer).

```

Comment:
```suggestion
Überprüfen Sie bei der Auswahl einer Beziehung zwischen Lagern und Lieferanten ID, die nicht eins zu eins übereinstimmt, das Verhalten der Einstellung [Lagerpuffer](#Lagerpuffer).
```

Reply:
Check that link above, it might actually need to be #stock-buffer and not #Lagerpuffer

Reply:
it should be #Lagerpuffer as this is the heading in the german guide